packages/docs/Next/Forms/InputFile.md

```
@@ -0,0 +1,41 @@
+# Input file
+
+### Default
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)